src/app/app-routing.module.ts

```diff
@@ -49,13 +49,13 @@ const routes: Routes = [
     canActivate: [AuthGuard]
   },
   {
-    path: '**',
```

Comment:
what if user enters the invalid route? have we handled it?

Reply:
we need to handle it through a different component like page not found

Reply:
using ** is not able navigate through auth library

Reply:
can we create a component for that and include a dummy picture similar to no data found ( we can ask products later for correct one) and one button below which says go back or go to login page something like that?

Reply:
+ once the user is logged in they should not be able to access the intro page (use route guard)